.github/ISSUE_TEMPLATE/2-feature-request.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Feature Request'
+labels: ['triage']
```

Comment:
Another place as the other triage discussion.

Reply:
done